presentation/backend/src/presentation-backend/NativePlatform.ts

```diff
@@ -194,7 +198,7 @@ export const createDefaultNativePlatform = (props: DefaultNativePlatformProps):
       const response = this._nativeAddon.handleRequest(db, options);
       cancelEvent?.addOnce(() => response.cancel());
       const result = await response.result;
-      return this.handleResult<string>(result);
+      return this.handleConvertedResult<Buffer, string>(result, (buffer) => buffer.toString());
```

Comment:
If some app upgrades their backend (or we upgrade GPB that they use) without updating their frontend, won't they start receiving `Buffer`s from presentation requests that they won't know what to do with?

Reply:
Our RPC didn't change - we send a string as before. It's just that the addon doesn't do the native string to `Napi::String` copy anymore.